### ISSUE TITLE
Too many open files error

### ISSUE THREAD

**Issue by boomanaiden154** [google/gematria#88]
```
Failed to annotate block: INTERNAL: Failed to create a pipe for interprocess communication between llvm-exegesis and the benchmarking subprocess: Too many open files
```

More investigation is needed. Probably an issue on the LLVM side, but opening here first in case there is some complicated interaction.

**Comment by boomanaiden154** [comment]
This is an upstream issue: https://github.com/llvm/llvm-project/issues/86583.

**Comment by boomanaiden154** [comment]
Closed by #115.
